File: 1421-FindNumbersWithEvenNumberOfDigits/1421-FindNumbersWithEvenNumberOfDigits.py
# Last updated: 6/17/2025, 10:25:53 PM
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        result = 0
        for i in nums:
            if (len(str(i)) % 2 == 0):
                result += 1
        return result
        

File: 1421-FindNumbersWithEvenNumberOfDigits/1421-FindNumbersWithEvenNumberOfDigits.java
// Last updated: 6/17/2025, 10:25:56 PM
class Solution {
    public int findNumbers(int[] nums) {
        int result = 0;
        for (int i = 0; i < nums.length; i++) {
            int numDigits = String.valueOf(nums[i]).length();
            if (numDigits % 2 == 0) {
                result++;
            }
        }
        return result;
    }
}
